tests/src/rules/order.js

```diff
@@ -2962,7 +2962,31 @@ context('TypeScript', function () {
               },
             ],
           }),
-        ],
+          isCoreModule('node:child_process') && isCoreModule('node:fs/promises') ? [
```

Comment:
node:fs/promises is commented out, why does this need to be a core module?

Reply:
I just copied that expression from this line in the `invalid` section: https://github.com/import-js/eslint-plugin-import/pull/2854/files#diff-fd60802036b3cdd3c033b19c132b29bdfbc608208d5bd5ea3b0e5ed2c1fe07d4R3214

Should the promises bit not be there?

Reply:
oh weird, lol then i guess it's fine as-is